redhat-cop/infra-ansible
# migrate depreciated include statements

Converted depreciated include and import statements from include to
include_* and import_* using 2.4 feature for playbooks and tasks.

closes #120

### What does this PR do?
Converts depreciated include and import statements to include_* and import_* using 2.4 feature for tasks and playbooks.

### How should this be tested?
Automated tests are preferred, but not always doable - especially for infrastructure. Include commands to run your new feature, and also post-run commands to validate that it worked. (please use code blocks to format code samples)

ansible-playbook --syntax-check ${playbook} 

### Is there a relevant Issue open for this?
https://github.com/redhat-cop/infra-ansible/issues/120
resolves #120 

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @redhat-cop/infra-ansible
